server/main.py

```diff
@@ -59,14 +62,17 @@ async def calculate_spectrum(payload: Payload):
             databank=payload.database,
             use_cached=True,
         )
+        if simulate_slit_value == True:
+            spectrum.apply_slit(payload.simulate_slit, "nm")
+        else:
+            pass
```

Comment:
I think we can simplify it a bit to just check if the value is None or not.
```suggestion
        if simulate_slit is not None:
            spectrum.apply_slit(payload.simulate_slit, "nm")
```